uui-core/services/ApiContext.ts

```diff
@@ -308,7 +308,7 @@ export class ApiContext extends BaseContext implements IApiContext {
                 }
```

Comment:
we need to return file with isError: true and error message from response on rejected promise